server/services/caldav/lib/calendar/calendar.syncCalendars.js

```diff
@@ -0,0 +1,31 @@
+const logger = require('../../../../utils/logger');
+
+/**
+ * @description Sync user calendars.
+ * @param {Array} calendars - External calendars.
+ * @param {Object} userId - Gladys calendar owner.
+ * @returns {Promise} Resolving with all calendars sync.
+ * @example
+ * syncCalendars(calendars, userId)
+ */
+async function syncCalendars(calendars, userId) {
```

Comment:
I'm not sure this function name really describe what it does. I see nothing about syncing here: no request is made in this function right?

Reply:
Yes it's true, I think it is more a function to `updateCalendars`. I am going to change also `syncCalendarEvents` to `updateCalendarEvents`.